custom_components/nest_protect/binary_sensor.py

```diff
@@ -38,19 +38,19 @@ class NestProtectBinarySensorDescription(
         key="co_status",
         name="CO Status",
         device_class=BinarySensorDeviceClass.CO,
-        value_fn=lambda state: state == 3,
+        value_fn=lambda state: state != 0,
```

Comment:
Can we move these changes to a separate PR?

Reply:
I've just copied the PR to keep it save and I'm not intedend to change it to much

Reply:
@Cruguah ok! I misunderstood :).

In that case I will see if I can work on top of this myself.

Reply:
> @Cruguah ok! I misunderstood :).
> 
> In that case I will see if I can work on top of this myself.

Would love to see where this these two ends meet, holding off my update to 2023.3.1
Thanks for all your work guys.